### DISCUSSION TITLE
Before and after downloading

### DISCUSSION THREAD

**Discussion by matt-franceschini-fw** [LironEr/bundlemon#110] [Ideas]
Another nice to have would be the ability to download before and after binaries. 

For example:
I see that my change added 1k to a file, but I have no indication of what it exactly is changed in that file. It would be nice to download the two files and do a quick diff.

Not sure if that is possible, but would a greatly appreciated feature.

**Comment by LironEr** [comment]
In order to do that the server will need to store all your app files, and currently it's not possible due to costs (as this is a free tool).

I will keep that in mind and maybe will add it as an option to store before and after files if you run your own server
